cc_corpus/download.py

```diff
@@ -67,34 +70,40 @@ def download_ranges(url: str,
             logging.error(f'Misc HTTP error for URL {url}: '
                           f'{r.status_code} - {r.text}; sleeping '
                           f'{orig_retry_left - retry_left}...')
-            time.sleep(orig_retry_left - retry_left)
+            time.sleep((orig_retry_left - retry_left) * delay_period)
             continue
     else:
         raise DownloadError(f'Could not download ranges from URL {url}.')
 
 
 def download_warc_ranges(
-    warc_file_name: str,
-    offsets_and_lengths: list[tuple[int, int]],
-    retry_left: int
+        warc_file_name: str,
```

Comment:
The default indent in Python is 4 spaces, not 8.

Reply:
PEP 8 style guide:

# Correct:
# Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)

# Wrong:
def long_function_name(
    var_one, var_two, var_three,
    var_four):
    print(var_one)

So I kept the 8 spaces of indentation, but moved the closing bracket to the end of the last parameter line, instead of starting a line of its own.